content/guides/models/app/features/panels/line-plot/smoothing.md

```diff
@@ -67,14 +75,16 @@ Consider using Gaussian Smoothing if your points are spaced unevenly on the x-ax
 
 The following image demonstrates how a running app looks like [in the app](https://wandb.ai/carey/smoothing-example/reports/W-B-Smoothing-Features--Vmlldzo1MzY3OTc#4.-running-average):
 
-{{< img src="/images/app_ui/running_average.png" alt="" >}}
-
-## Exponential Moving Average (Deprecated)
+{{< img src="/images/app_ui/running_average.png" alt="Demo of running average smoothing" >}}
 
-> The TensorBoard EMA algorithm has been deprecated as it cannot accurately smooth multiple lines on the same chart that do not have a consistent point density (number of points plotted per unit of x-axis).
+## Exponential Moving Average
 
-Exponential moving average is implemented to match TensorBoard's smoothing algorithm. The range is 0 to 1. See [Exponential Smoothing](https://www.wikiwand.com/en/Exponential_smoothing) for background. There is a debias term added so that early values in the time series are not biases towards zero.
+The Exponential Moving Average (EMA) smoothing algorithm is implemented to match TensorBoard's smoothing algorithm. The range is 0 to 1. See [Exponential Smoothing](https://www.wikiwand.com/en/Exponential_smoothing) for background. A debias term is added so that early values in the time series are not biases towards zero.
 
+EMA smoothing is applied by the W&B App after sampling and bucketing. In general, it is less accurate than [TWEMA smoothing]({{< relref "#time-weighted-exponential-moving-average-default" >}}). However, EMA is used instead of TWEMA in the following situations:
+- Sampling
+- Grouping
+- non-monotonic x-axes
```

Comment:
this list should also include 

- expressions
- time-based x axes

Reply:
Done